package.json

```diff
@@ -82,6 +82,7 @@
     "fs-extra": "^0.23.1",
     "glob": "^7.1.2",
     "google-auto-auth": "^0.7.2",
+    "grpc": "^1.21.1",
```

Comment:
does `firebase-tools` itself need this dependency or should this be part of the `package.json` of the individual integration tests?